front-end/src/app/transactions/transaction-list/transaction-list.component.ts

```diff
@@ -89,6 +109,27 @@ export class TransactionListComponent extends TableListBaseComponent<Transaction
     const reportId = this.activatedRoute.snapshot.params['reportId'];
     return { report_id: reportId };
   }
+
+  public onRowActionClick(action: TableAction, transaction: Transaction) {
+    action.action(transaction);
+  }
+
+  public forceItemize(transaction: Transaction): void {
+    transaction.force_itemized = true;
+    this.updateItem(transaction);
+  }
+
+  public forceUnitemize(transaction: Transaction): void {
+    transaction.force_itemized = false;
+    this.updateItem(transaction);
+  }
+
+  public updateItem(item: Transaction) {
+    this.itemService.update(item).subscribe(() => {
```

Comment:
probably want a takeUntil(this.destroy$)

Reply:
I opted for 

```
take(1),
takeUntil(this.destroy$)
```

Which will resolved either when the first is received or component is destroyed